### DISCUSSION TITLE
Understanding versions/release

### DISCUSSION THREAD

**Discussion by ghost** [Azure/AKS#2233] [Q&A]
How does one tell which Kube Version a release contains? Take 'Release 2021-03-22', I see no mention as to which version of Kube that contains. Is there a page that maps out this information?

**Comment by chrisbear** [comment]
https://docs.microsoft.com/en-gb/azure/aks/tutorial-kubernetes-upgrade-cluster?tabs=azure-cli

az aks get-upgrades --resource-group myResourceGroup --name myAKSCluster
